.eslintrc.js

```diff
@@ -1,3 +1,16 @@
 module.exports = {
-  "extends": "google"
-};
\ No newline at end of file
+  'extends': 'google',
+  'parser': 'babel-eslint',
+  'env': {
```

Comment:
it is not a good thing to have `browser` environment enabled in the back end side and the `node` environment in the front end side!

try to fix this so `browser` is only enabled in the front end codes and `node` is only enabled in the back end codes

https://eslint.org/docs/user-guide/configuring#specifying-environments